src/common/hooks/useScrollIntoView.js

```diff
@@ -0,0 +1,7 @@
+export default function useScrollIntoView(){
```

Comment:
Wouldn't we also need import useEffect here as well at the top? Otherwise I see that useEffect is not defined in index.js